Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -346,6 +351,12 @@ namespace O3DE::ProjectManager
         m_projectsFlowLayout->update();
     }
 
+    void ProjectsScreen::HandleProjectDirectoryChanged(const QString& /*path*/)
+    {
+        // QFileWatcher automatically stops watching the path if it was removed so we will just refresh our view
+        ResetProjectsContent();
+    }
```

Comment:
This will reset the project content when any file is added or removed in the project folder.  Can we be more selective and just monitor `project.json` in the folder and ignore all the other changes?